napalm_ansible/modules/napalm_install_config.py

```diff
@@ -288,7 +286,7 @@ def main():
 
     try:
         if get_diffs:
-            diff = device.compare_config().encode('utf-8')
```

Comment:
I can't see a good reason to encode this to a byte-string? Anyone see why we should be doing that?

Reply:
I don't.